approval_frame/forms.py

```diff
@@ -17,3 +18,22 @@ class RegistrationFormCustom(RegistrationFormUniqueEmail):
         error_messages={'invalid': _("This value may contain only letters, "
             + "numbers and ./+/-/_ characters.")}
     )
+
+class PasswordResetFormCustom(PasswordResetForm):
```

Comment:
Ah, I understand. Nice article. I guess the user in that case should remember the email ID with which he created the account. 
I had noticed that Github does indicate if the email address is a registered one as a part of its validation, so thought that our application should have the same. 
I'll make the required changes though ! Thanks